sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c

```diff
@@ -352,13 +352,31 @@ static int _fsm_call_close(gnrc_tcp_tcb_t *tcb)
 /**
  * @brief FSM handling function for forcefull connection teardown sequence.
  *
- * @returns   -EOPNOTSUPP. Currently not implemented.
+ * @param[in,out] tcb   TCB holding the connection information.
+ *
+ * @returns   Zero on success.
  */
-static int _fsm_call_abort(void)
+static int _fsm_call_abort(gnrc_tcp_tcb_t *tcb)
 {
+    gnrc_pktsnip_t *out_pkt = NULL;     /* Outgoing packet */
+    uint16_t seq_con = 0;               /* Sequence number consumption of outgoing packet */
```

Comment:
scope of both variables can be reduced, move into `if` below.

Docu of such internal variables is not necessary, especially when obvious 😉